### DISCUSSION TITLE
如何启用GPU推理？

### DISCUSSION THREAD

**Discussion by C-zZz-K** [svc-develop-team/so-vits-svc#137] [Q&A]
![image](https://user-images.githubusercontent.com/65231499/230818677-cf2daa30-9c3b-4fd6-932c-c18dde3933f6.png)
如何启用GPU推理？我记得以前可以直接GPU推理，为什么新版本找不到了？之前好像在哪看到过解决办法，但是找不到了

**Comment by Miuzarte** [comment]
更新源码之后检查torch版本是不是不带cuda
`pip show torch`
带cuda的版本号应类似于`1.13.1+cu117`
